lib/caqti_response_sig.mli

```diff
@@ -79,4 +81,9 @@ module type S = sig
 
       {b Note.} Do not make nested queries in the callback to this function.
       Cf. {!fold_s}. *)
+
+  val to_stream : ('b, 'm) t -> ('b, [> Caqti_error.retrieve] as 'e) result stream
+  (** [to_stream resp] returns a stream whose elements are the decoded rows returned
+      by [resp].
+      If an error is encountered it will be the last element of the stream. *)
```

Comment:
Please break lines before column 81, for the signature I prefer right after the colon in this case.